README.md

```diff
@@ -88,6 +88,7 @@ CHANGELOG
 ### 2.4.16
 
 * Fix handling of floating-point precision near the aligned case that used to result in error from libphoebe. [#965]
+* Fix support for python 3.13, remove official support for python 3.8. [#968]
```

Comment:
you're saying remove 3.8 here, but in the toml file we have >=3.8; one of them needs to be updated.

Reply:
ah, good catch, I meant removing support for 3.7.  I'll update that here.